src/tracing/errors.ts

```diff
@@ -40,6 +40,21 @@ export const filterKnownErrors: Required<ClientOptions>['beforeSend'] = (event:
     // If the error is based on a user rejecting, it should not be considered an exception.
     if (didUserReject(error)) return null
 
+    /*
+     * This ignores 499 errors, which are caused by Cloudflare when a request is cancelled.
+     * CF claims that some number of these is expected, and that they should be ignored.
+     */
+    if (error.message.match(/Loading chunk \d+ failed\. \(error: .+\.chunk\.js\)/)) {
```

Comment:
This will only catch js chunk errors. You should catch all errors (eg css, etc).

```suggestion
    if (error.message.match(/Loading chunk \d+ failed\. \(error: .+\.chunk\.\w{2+}\)/)) {
```

Reply:
That won't actually be sufficient because the error message is slightly different for example. So I'd like to leave that for a separate PR.